src/desktop/apps/home/templates/hero_unit.jade

```diff
@@ -1,6 +1,6 @@
 .home-hero-unit(
   class= 'home-hero-unit' + _s.dasherize(_s.humanize(heroUnit.mode)) + (i == 0 ? ' home-hero-unit-active' : '')
-  style="background-image: url('#{heroUnit.background_image_url}')"
+  style="background-image: url('#{resize(heroUnit.background_image_url, { width: 2000 })}')"
```

Comment:
We pass in `width: 2000` to the `resize` function, which means we aren't actually resizing the images but just changing the quality.